features/cors.yml

```
@@ -0,0 +1,18 @@
+name: CORS
+description: Cross-Origin Resource Sharing (CORS) is an HTTP-header based mechanism that allows a server to indicate any origins (domain, scheme, or port) other than its own from which a browser should permit loading resources.
```

Just to make something explicit here: if we're not going to reference the abbreviation again in the description text, we should move the abbreviation to an "also known as". But if you need it for the rest of the text to make sense, you'd still do the parenthetical (e.g., "The File Transfer Protocol (FTP) sends a file from one host to another. FTP is a predecessor to the web.")

```suggestion
description: Cross-Origin Resource Sharing is an HTTP-header based mechanism that allows a server to indicate any origins (domain, scheme, or port) other than its own from which a browser should permit loading resources. Also known as CORS.
```